fetch-lorempixel-images.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# Demo use only. Images from lorempixel, all rights [p]reserved.
+for i in `seq 10` ; do curl -s -H "Accept: image/jpg" http://lorempixel.com/400/400/nature/$i/ >$i.jpg ; done
```

Comment:
This script is in the root folder, the committed images are in `lorempixel`. Likely it should either put the images into `lorempixel/$i.jpg` or should be moved into that folder to fetch into the current directory from where it is run by default.